client/src/main/java/org/camunda/bpm/client/topic/TopicSubscriptionBuilder.java

```diff
@@ -96,6 +98,20 @@
    */
   TopicSubscriptionBuilder processDefinitionVersionTag(String processDefinitionVersionTag);
 
+  /**
+   * @param processVariables to search for external tasks that are supposed to be
+   *                         fetched and locked
+   * @return the builder
+   */
+  TopicSubscriptionBuilder processVariablesContain(Map<String, Object> processVariables);
```

Comment:
```suggestion
  TopicSubscriptionBuilder processInstanceVariableEquals(Map<String, Object> processVariables);
```
Let's use the naming of the engine implementation.

Reply:
Hi @tasso94 , I originally did have the same idea, but then what stopped me was, the fact that functionally, the implementation does not look to match ALL of the variables supplied but instead passed when ANY of the variables is matched. However, if you still think this is a non-issue, I shall proceed and rename to match the engine implementation. Cheers

Reply:
You are right! What about `processVariableIn`? We have this naming convention at many places for the OR case.

Reply:
Sounds good, you meant `processInstanceVariableIn` I believe?

Reply:
I think we don't use the term "Process Instance Variable" so frequently. "Process Variable" is more common.

Reply:
Sure and Thanks